.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: mbgl/ci:r4-linux-clang-3.9
```

Comment:
Does this image have awscli pre-installed? We should use one that does, so we can remove that from `dependencies.sh`.